node/src/components/consensus/era_supervisor.rs

```diff
@@ -188,6 +188,7 @@ where
         // The number of rounds after which a block reward is paid out.
         // TODO: Make this configurable?
         let reward_delay = 8;
+        // TODO: The initial round length should be the observed median of the switch block.
```

Comment:
What does it mean _observed median of the switch block_? Did you mean _observed median of votes voting on a switch block_?

Reply:
Ah, I understand now – it's the median of round exp observed by the switch block.